sqlglot/generator.py

```diff
@@ -130,7 +130,7 @@ def __init__(
         self.time_trie = time_trie
         self.pretty = pretty if pretty is not None else sqlglot.pretty
         self.configured_pretty = self.pretty
-        self.identifier = identifier or '"'
+        self.identifier = identifier or ('"', '"')
```

Comment:
what do you think if we change it to

self.identifier_start, self.identifier_end = identifier or ('"', '"') that's probably more clear and then in 401 we can just use those